jdaviz/configs/mosviz/plugins/slit_overlay/slit_overlay.py

```diff
@@ -78,11 +77,11 @@ def place_slit_overlay(self, *args, **kwargs):
         self.remove_slit_overlay()
 
         # Get data from relevant viewers
-        image_data = self.app.get_viewer("image-viewer").data()
+        image_data = self.app.get_viewer("image-viewer").state.reference_data
```

Comment:
I don't know why original code wanted `image_data[0]` specifically. I just assume `reference_data` is the same thing. If not, please let me know. Thanks!

Reply:
I think this was an assumption about the data format, so I think you are doing the right thing here!

Reply:
Yeah, this works.